be/src/runtime/routine_load/data_consumer_group.cpp

```diff
@@ -181,7 +180,7 @@ Status KafkaDataConsumerGroup::start_all(StreamLoadContext* ctx) {
                 VLOG(3) << "consume partition[" << msg->partition() << " - " << msg->offset() << "]";
             } else {
                 // failed to append this msg, we must stop
-                LOG(WARNING) << "failed to append msg to pipe. grp: " << _grp_id;
+                LOG(WARNING) << "failed to append msg to pipe. grp: " << _grp_id << ", errmsg=" << st.get_error_msg();
```

Comment:
LOG(WARNING) << "failed to append msg to pipe. grp: " << _grp_id << ", errmsg=" << st;